### ISSUE TITLE
Modify cdf_units() to be able to take only some custom units

### ISSUE THREAD

**Issue by dstansby** [heliopython/heliopy#471]
At the moment `cdf_units` either extracts all the keys from the CDF if `keys=None`, or only the keys defined in `keys` if it is a dict.

It would be good to use `keys` as a dictionary of some custom units, while still doing automatic unit extraction on keys that are in the CDF but not the `keys` argument.

**Comment by abotiamnot** [comment]
Yep, I've been thinking of making it that way that the user can also provide the units.
